admin_guide/manage_authorization_policy.adoc

```diff
@@ -0,0 +1,283 @@
+= Managing Authorization Policy
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+You can use the administrator CLI to manage
```

Comment:
@bfallonf I think an "Administrative CLI" topic is warranted under the CLI Reference directory. Been thinking of making one for a while, too, actually. I'll follow up later with something, and we could update references like these to link to that for more context.